gempython/tools/amc_user_functions_uhal.py

```diff
@@ -172,3 +172,6 @@ def blockL1A(device):
 
 def enableL1A(device):
     writeRegister(device, "GEM_AMC.TTC.CTRL.L1A_ENABLE", 0x1)
+
+def getAMCL1Acount(device):
+    return readRegister(device, "GEM_AMC.TTC.CMD_COUNTERS.L1A")
```

Comment:
the problem here is that we don't have classes in python while we do in xDAQ. Also python doesn't support methods overloading. I can see two solutions, both of which I don't really like...
* I can make the functions for AMC and for OH with the same name and call them as module_name.function_name, but this might be ambigous
* I can set default link value to `-1` and use it for AMC, and if the link value provided, use it for OH - but this is also very artificial and won't make code very clear...